### PR TITLE
Issue 644 fix qhy central fov

### LINKED ISSUE #644: QHY observations using central image region and half-rate tracking mode results in object leaving the chip
A long (~3 hr)  DeltaRho/QHY [obs](https://observe.lco.global/requests/3573885) of a (moderately) Speedy Rock (Tm) was attempted using the half-rate tracking option. This should have been split into chunks so that the rock doesn't scuttle off the chip but it wasn't resulting in it leaving the chip after ~0.7hr.
Looking at the code in `get_exposure_bins` called from `split_configs` it seems that there is an implicit assumption that a different binning implies a different FOV which was true for Sinistro but not for QHY:
```
print(f"site_code= {params['site_code']}, bin_mode= {params.get('bin_mode', None)}")
site_code= 0M4, bin_mode= central30_1x1
get_sitecam_params(params['site_code'], params.get('bin_mode', None))

Out[107]: 
{'site_code': '0M4',
 'setup_overhead': 90.0,
 'exp_overhead': 6,
 'pixel_scale': 0.7386,
*** 'fov': 117.9, ***
 'max_exp_length': 300.0,
 'alt_limit': 15.0}
```

### PR DESCRIPTION
**Pull request by talister**
tldr; extra underscore in a mode name in one place.... took the opportunity to add Moar Tests and remove some old 0.4m+SBIG bits.
Fixes #644

### CODE REVIEW

**Review by jchate6 — APPROVED** [review]
ooooooooof that underscore.
I'd complain to management. :D

Good catch, looks good.

**Review by carrholt — APPROVED** [review]
wow, good catch! new tests also look good.
